### DISCUSSION TITLE
Is there a way or example on how to override all styles or a headlessUI version ?

### DISCUSSION THREAD

**Discussion by VaZark** [Comcast/react-data-grid#2912] [Q&A]
The component is pretty neat and complete. However, I am unable to find examples on how to override columns, individual cells etc. Is there any way or example to completely overhaul the theming of the grid? It would be nice to have a headless version of the component available.

**Comment by iamravisingh** [comment]
@VaZark you can directly use the class of component and override it 
for header you can you **.rdg-header-row** for row **.rdg-row**
cell inside row **.rdg-row > .rdg-cell** same for header. Also, there are classed defined for certain area. I would recommend to fork the project and have look into the code
